File: fixtures/scrapy_output/get_with_data.py
from scrapy import Request, Spider


class MySpider(Spider):
    name = 'myspider'

    def start_requests(self):
        headers = {
            'X-Api-Key': '123456789',
        }
        yield Request(
            'https://synthetics.newrelic.com/synthetics/api/v3/monitors?test=2&limit=100&w=4',
            callback=self.parse,
            headers=headers,
        )

    def parse(self, response):
        pass

File: test/fixtures/php-guzzle/get_with_data.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->get('http://localhost:28139/synthetics/api/v3/monitors', [
    'query' => [
        'test' => '2',
        'limit' => '100',
        'w' => '4'
    ],
    'headers' => [
        'X-Api-Key' => '123456789'
    ]
]);
